docs/website/docs/reference/optimization-options.md

```diff
@@ -18,7 +18,7 @@ These flags can be passed to the:
 
 ## High level program optimizations
 
-### Constant evaluation (`--iree-opt-const-eval` (off))
+### Constant evaluation (`--iree-opt-const-eval` (on))
```

Comment:
Could we say this fixes https://github.com/openxla/iree/issues/12033, or are there other flags that are still default-disabled? I had been trying both `--iree-opt-const-expr-hoisting` and `--iree-opt-const-eval` as ways to take pressure off of `FusionOfTensorOps`, and it sounds like you saw some great improvements with this change:

> Drops compilation time of h2ogpt from ~45m to 2m47s. More optimization is possible.
JIT'ing globals now takes 52s for this model (vs ~42m with the Linalg const evaler).

Reply:
Yes, I think so. What I heard from @MaheshRavishankar is that 99% of the issue is from transpose canonicalization patterns. If we enable const-eval for the case, we fix the issue.

Reply:
I will remove the FusionOfTensorOps canonicalizations in a followup once fully enabled. Some more work needed beyond this.